packages/core/addon/components/navi-vis-c3-chart.js

```diff
@@ -169,8 +169,8 @@ export default C3Chart.extend({
    * @returns {Void}
    */
   _teardownChart() {
-    get(this, 'chart').destroy();
-    set(this, '_chart', null);
+    //get(this, 'chart').destroy();
```

Comment:
```suggestion

```

Reply:
Is the cleanup logic happening elsewhere now?

Reply:
This was also called on attr update

Reply:
this absolutely breaks things when you try to destroy from attr update